source/PreferencesPanel.cpp

```diff
@@ -584,26 +585,43 @@ void PreferencesPanel::DrawPlugins()
 	const Color &medium = *GameData::Colors().Get("medium");
 	const Color &bright = *GameData::Colors().Get("bright");
 
+	const Sprite *box[2] = { SpriteSet::Get("ui/unchecked"), SpriteSet::Get("ui/checked") };
+
 	const int MAX_TEXT_WIDTH = 230;
 	Table table;
 	table.AddColumn(-115, {MAX_TEXT_WIDTH, Truncate::MIDDLE});
-	table.SetUnderline(-120, 120);
+	table.SetUnderline(-120, 100);
 
 	int firstY = -238;
-	table.DrawAt(Point(-130, firstY));
+	// Table is at -110 while checkbox is at -130
+	table.DrawAt(Point(-110, firstY));
 	table.DrawUnderline(medium);
 	table.Draw("Installed plugins:", bright);
 	table.DrawGap(5);
 
 	const Font &font = FontSet::Get(14);
+
 	for(const auto &plugin : GameData::PluginAboutText())
 	{
 		pluginZones.emplace_back(table.GetCenterPoint(), table.GetRowSize(), plugin.first);
 
 		bool isSelected = (plugin.first == selectedPlugin);
 		if(isSelected || plugin.first == hoverPlugin)
 			table.DrawHighlight(back);
-		table.Draw(plugin.first, isSelected ? bright : medium);
+		bool pluginEnabled = Plugins::IsEnabled(plugin.first);
+		Point checkboxPos = table.GetRowBounds().TopLeft();
+		checkboxPos.X() -= font.Height() / 2;
+		checkboxPos.Y() += table.GetRowBounds().Height() / 2;
+
+		SpriteShader::Draw(box[pluginEnabled], checkboxPos);
+		Point zoneDimension = Point(20., table.GetRowBounds().Height() * 2. / 3.);
+		Point zoneOffset = checkboxPos + Point(0., table.GetRowBounds().Height() / 3);
```

Comment:
Lots of calls to `GetRowBounds().Height()` suggests you can improve this

Reply:
Yeah, I was actually struggling with this clickable zone.  It's still not perfect.  I tried making everything relative to the table row.  I can DRY it up some.